src/policy/fees.cpp

```diff
@@ -374,15 +374,17 @@ double TxConfirmStats::EstimateMedianVal(int confTarget, double sufficientTxVal,
         failBucket.leftMempool = failNum;
     }
 
-    LogPrint(BCLog::ESTIMATEFEE, "FeeEst: %d %s%.0f%% decay %.5f: feerate: %g from (%g - %g) %.2f%% %.1f/(%.1f %d mem %.1f out) Fail: (%g - %g) %.2f%% %.1f/(%.1f %d mem %.1f out)\n",
-             confTarget, requireGreater ? ">" : "<", 100.0 * successBreakPoint, decay,
-             median, passBucket.start, passBucket.end,
-             100 * passBucket.withinTarget / (passBucket.totalConfirmed + passBucket.inMempool + passBucket.leftMempool),
-             passBucket.withinTarget, passBucket.totalConfirmed, passBucket.inMempool, passBucket.leftMempool,
-             failBucket.start, failBucket.end,
-             100 * failBucket.withinTarget / (failBucket.totalConfirmed + failBucket.inMempool + failBucket.leftMempool),
-             failBucket.withinTarget, failBucket.totalConfirmed, failBucket.inMempool, failBucket.leftMempool);
-
+    if (passBucket.totalConfirmed + passBucket.inMempool + passBucket.leftMempool != 0 && failBucket.totalConfirmed + failBucket.inMempool + failBucket.leftMempool != 0) {
```

Comment:
Can any of this sums be zero? I guess no otherwise this was raised before. Just assert? Same below in wallet.cpp.

Reply:
Yes they can be zero: this UB is triggered when running the functional tests.

As said in the PR description:

> UBSAN in Travis running the test suite would have catched all three of these.

:-)